example-mlem-get-started/generate.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+# See https://dvc.org/doc/tutorials/get-started
+
+# Setup script env:
+#   e   Exit immediately if a command exits with a non-zero exit status.
+#   u   Treat unset variables as an error when substituting.
+#   x   Print commands and their arguments as they are executed.
+set -eux
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+REPO_NAME="example-mlem-get-started"
+
+BUILD_PATH="$HERE/build"
+
+pushd $BUILD_PATH
+if [ ! -d "$BUILD_PATH/.venv" ]; then
+  exit 1
+  virtualenv -p python3 .venv
+  export VIRTUAL_ENV_DISABLE_PROMPT=true
+  source .venv/bin/activate
+  echo '.venv/' > .gitignore
+  pip install gitpython
+  pip install "git+https://github.com/iterative/mlem#egg=mlem[all]" --use-deprecated=legacy-resolver
+  pip install -r $HERE/code/src/requirements.txt
+fi
+popd
+
+source $BUILD_PATH/.venv/bin/activate
+
+REPO_PATH="$HERE/build/$REPO_NAME"
+
+if [ -d "$REPO_PATH" ]; then
+  echo "Repo $REPO_PATH already exists, please remove it first."
+  exit 1
+fi
+
+TOTAL_TAGS=15
+STEP_TIME=100000
+BEGIN_TIME=$(( $(date +%s) - ( ${TOTAL_TAGS} * ${STEP_TIME}) ))
+export TAG_TIME=${BEGIN_TIME}
+export GIT_AUTHOR_DATE=${TAG_TIME}
+export GIT_COMMITTER_DATE=${TAG_TIME}
+tick(){
+  export TAG_TIME=$(( ${TAG_TIME} + ${STEP_TIME} ))
+  export GIT_AUTHOR_DATE=${TAG_TIME}
+  export GIT_COMMITTER_DATE=${TAG_TIME}
+}
+
+export GIT_AUTHOR_NAME="Mikhail Sveshnikov"
+export GIT_AUTHOR_EMAIL="mike0sv@gmail.com"
+export GIT_COMMITTER_NAME="$GIT_AUTHOR_NAME"
+export GIT_COMMITTER_EMAIL="$GIT_AUTHOR_EMAIL"
+
+
+
+mkdir -p $REPO_PATH
+pushd $REPO_PATH
+
+git init -b main
+cp $HERE/code/README.md .
+cp $HERE/.gitattributes .
+git add .
+tick
+git commit -m "Initialize Git repository"
+git tag -a "0-git-init" -m "Git initialized."
+
+mlem init
+tick
+git add .mlem/config.yaml
+git commit -m "Initialize MLEM project"
+git tag -a "1-mlem-init" -m "MLEM initialized."
+
+
+cp $HERE/code/src/requirements.txt .
+cp $HERE/code/src/prepare.py .
+git add .
+python prepare.py
+git add .mlem/dataset
+tick
+git commit -m "Create data stage"
+git tag -a "2-prepare-stage" -m "Data created."
+
+
+cp $HERE/code/src/train.py .
+python train.py
+git add .mlem/model train.py
+tick
+git commit -m "Create train stage"
+git tag -a "3-train-stage" -m "Model trained."
+
+
+cp $HERE/code/src/evaluate.py .
+python evaluate.py
+git add metrics.json evaluate.py
+tick
+git commit -m "Evaluate model"
+git tag -a "4-eval-stage" -m "Metrics calculated"
+
+git rm -r --cached .mlem/model/rf .mlem/dataset/train.csv .mlem/dataset/test_x.csv .mlem/dataset/test_y.csv
+git commit -m "stop tracking data"
```

Comment:
why is it called "stop tracking data"?

Reply:
do we need this extra commit?

Reply:
but I don't think we need an extra commit ... could you clarify `this one is suggested by DVC itself`?

Reply:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/5938179/164321310-62d813af-d703-41d1-80ee-b9f5ffa6d0d5.png">
without those two lines. But it works without commit, I checked. Just thought commit is necessary since DVC suggested it